package.json

```diff
@@ -8,6 +8,7 @@
     "start": "node dist/app",
     "lint": "eslint src test",
     "test": "npm run lint && mocha --require babel-register test/**/*-test.js",
+    "testcafe": "testcafe chrome test/functional/test.js",
```

Comment:
Let's use a glob pattern to make it possible to start all functional tests if they will be added, e.g. `test/functional/**/*.js`